### DISCUSSION TITLE
Welcome to NKC Discussions!

### DISCUSSION THREAD

**Discussion by Creep69** [Creep69/NKC#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mczero80** [comment]
Hi,
do you also have the CAS-NEO layout as a gerber file?

Thank you very much!

**Reply by Creep69** [reply]
Yes, I have. It's just not (yet?) in the repository. Was done by derInder.

**Comment by mczero80** [comment]
Would be nice if you could add it into this repo!

Btw., I had a problem uploading most of the gerber files to jlcpcb. The outline files weren't recognized and therefore the board sizes could not be loaded. As a workaround, I installed the Sprint Demo application and upon loading the sprint files, I could get the correct sizes. Then I could set them manually.

The boards that had no Sprint file had a functioning outline file in the gerber zip file, the others not.

Maybe it is just jlcpcb, don't know.

I look forward building my first NDR Klein Computer!

**Comment by Creep69** [comment]
Strange, I ordered most of the PCBs at JLCPCB. If you email me, I can double check and send you the 
gerbers. I don't want to put all of them here without asking the authors. public at bireve dot de
NKC is a great system. I've build Z80, 6502, i8080, 68008, 68000, 68020 and more to come.

**Reply by mczero80** [reply]
Strange indeed. I wrote you an e-mail!
